tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,269 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b18c8d8c",
+   "metadata": {},
+   "source": [
+    "### Launch EVA DB\n",
+    "Run the command `python eva.py` in the server where you want to deploy EVA"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3c5b2a6f",
+   "metadata": {},
+   "source": [
+    "### Establish Connection With Eva DB"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "62e83a0b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.insert(0,'..')\n",
+    "from src.server.db_api import connect"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "b78a43e3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import nest_asyncio\n",
+    "# nest_asyncio.apply()\n",
```

Comment:
Any reason why this is commented? When this is commented, I am facing the `event loop is already running` issue when I run the notebook on `ada-02`. The error goes away when I uncomment this.

Reply:
I was not sure if this is only showing up in my environment. Ideally, we should fix the root issue. Anyways, I'll uncomment for now.